apps/hermes/server/README.md

```diff
@@ -21,9 +21,10 @@ To set up and run a Hermes node, follow the steps below:
    using [Beacon](https://github.com/pyth-network/beacon), a highly available rewrite for spy, for production purposes.
 1. **Install Rust nightly-2023-07-23**: If you haven't already, you'll need to install Rust. You can
    do so by following the official instructions. Then, run the following command to install the required
-   nightly version of Rust:
+   nightly version of Rust and set it as the default:
```

Comment:
I think this repo doesn't have `rust-toolchain` file and the issue is rooted in this. I think instead of asking user to set this specific rust version to repo, let's add the `rust-toolchain` file so the correct version is automatically picked up by the compiler.

Reply:
It is actually there in the `server/` dir, but for some reason cargo wouldnt use the right toolchain until I set it as default, not sure why

Reply:
I'll go ahead and remove though, rust-toolchain does indeed solve this